src/Razor/src/Microsoft.AspNetCore.Razor.VSCode/src/Document/RazorDocumentManager.ts

```diff
@@ -95,11 +95,11 @@ export class RazorDocumentManager implements IRazorDocumentManager {
 
             this.closeDocument(document.uri);
         });
-        this.serverClient.onRequest(
+        this.serverClient.onNotification(
```

Comment:
Good catch, it didn't occur to me that moving closer to the spec might cause issue 😆